packages/teleport-cli/Readme.md

```diff
@@ -0,0 +1,63 @@
+## @teleporthq/teleport-cli
+
+### Clone a Project
+
+```shell
+teleport clone --link <link-to-project> --path <sys-path>
+
+teleport clone -l https://repl.teleporthq.io/project/2c5878e9-1bbc-4f7c-a27a-b8822dd39571
+
+teleport clone --link https://teleport-gui-git-development-teleport-team.vercel.app/projects/new-project-zan7
+```
+
+### Clone a Component
+
+```shell
+teleport clone --link <link-to-component> --path <sys-path>
+
+teleport clone --link https://repl.teleporthq.io/\?uidlLink\=35c82f1c-d272-4f8b-8a3f-5fa44702a1e6\&flavor\=React\&style\=CSS-Modules
```

Comment:
Do we need to escape characters in the URL?

Reply:
Not needed, this is just i copied from my terminal as is. You can just copy from `url` bar